extensions/wikia/WikiaWhiteList/WikiaWhiteList.php

```diff
@@ -93,6 +94,19 @@ function wfWhiteListParse($text) {
 							if (count($m) > 0) {
 								$text = preg_replace(sprintf(REGEX_PARSE_WHITELIST,"rel","nofollow"), "", $text);
 								$text = preg_replace(sprintf(CLASS_PARSE_WHITELIST,"external","(text|free|autonumber)"), "class=\"$1\"", $text);
+
+								$parsed = @parse_url( $lhref );
```

Comment:
I'm not sure silencing errors is needed. In case of malformed links `parse_url ` should simply return `FALSE`.